packages/fluent-theme/src/components/theme/Theme.module.css

```diff
@@ -14,19 +16,27 @@
   --webchat-colorNeutralForegroundDisabled: var(--colorNeutralForegroundDisabled, #bdbdbd);
 
   --webchat-colorNeutralBackground1: var(--colorNeutralBackground1, #ffffff);
+  --webchat-colorNeutralBackground1Hover: var(--colorNeutralBackground1Hover, #f5f5f5);
   --webchat-colorNeutralBackground3: var(--colorNeutralBackground3, #f5f5f5);
   --webchat-colorNeutralBackground4: var(--colorNeutralBackground4, #f0f0f0);
   --webchat-colorNeutralBackground5: var(--colorNeutralBackground5, #ebebeb);
 
   --webchat-colorTransparentBackground: var(--colorTransparentBackground, rgba(0, 0, 0, 0.4));
 
+  --webchat-colorNeutralBackgroundDisabled: var(--colorNeutralBackground1Disabled, #f0f0f0);
   --webchat-colorNeutralBackground1Disabled: var(--colorNeutralBackground1Disabled, #f0f0f0);
-  --webchat-colorNeutralBackground1Hover: var(--colorNeutralBackground1Pressed, #f0f0f0);
   --webchat-colorNeutralBackground1Pressed: var(--colorNeutralBackground1Pressed, #e0e0e0);
 
+  --webchat-colorNeutralBackground1Disabled: var(--colorNeutralBackground1Disabled, #f0f0f0);
+  --webchat-colorNeutralBackground1Hover: var(--colorNeutralBackground1Hover, #f0f0f0);
+  --webchat-colorNeutralBackground1Pressed: var(--colorNeutralBackground1Pressed, #e0e0e0);
```

Comment:
This seems incorrect:
 - the `--webchat-colorNeutralBackground1Disabled` is repeated two times
 - the `--webchat-colorNeutralBackgroundDisabled` is linked to `--colorNeutralBackgroundDisabled`
 - the ` --webchat-colorNeutralBackground1Hover` repeats two times (see line 19

Reply:
We badly need a way to generate this. 🤣

Reply:
:100:

Reply:
I think this time, it should be better.